ledger/src/blocktree.rs

```diff
@@ -4071,4 +4079,59 @@ pub mod tests {
         drop(ledger);
         Blocktree::destroy(&ledger_path).expect("Expected successful database destruction");
     }
+
+    #[test]
+    pub fn test_persist_transaction_status() {
+        let blocktree_path = get_tmp_ledger_path!();
+        {
+            let blocktree = Blocktree::open(&blocktree_path).unwrap();
+            let transaction_status_cf = blocktree.db.column::<cf::TransactionStatus>();
+
+            // result not found
+            assert!(transaction_status_cf
+                .get((0, Signature::default()))
+                .unwrap()
+                .is_none());
```

Comment:
The nice thing about using `assert_eq!` here and in a couple places below is that if != the error message will include the LHS value, which can be help make for a quicker debug session
```rust
assert_eq!(transaction_status_cf.get((0, Signature::default())).unwrap(),None);
```

Reply:
I need to run right now so I only have time to (1) merge the change after successful CI or (2) make the change and let someone else merge this PR... What should I do?

Reply:
My comment was just a nit, no need to block this PR from landing on it

Reply:
sounds good, will queue this for a subsequent PR